docs/kyma-environment-broker/02-01-architecture.md

```diff
@@ -2,23 +2,19 @@
 
 The diagram and steps describe the Kyma Environment Broker (KEB) workflow and the roles of specific components in this process:
 
-![KEB diagram](./assets/keb-architecture.svg)
+![KEB diagram](./assets/keb-arch.svg)
 
 1. The user sends a request to create a new cluster with [Kyma Runtime](https://github.com/kyma-incubator/compass/blob/master/docs/compass/02-01-components.md#kyma-runtime).
 
-    a. The user sends a request to KEB through [Istio VirtualService](https://istio.io/docs/reference/config/networking/virtual-service/).
-
-    b. Istio redirects the request to the [Ory Oathkeeper](https://www.ory.sh/oathkeeper/docs/), which authorizes the request.
-
-    c. If the authorization ends with success, the request is redirected to KEB.
-
 2. KEB proxies the request to create a new cluster to the Runtime Provisioner component.
 
-3. Provisioner registers a new cluster in the Director component.
+3. Provisioner creates a new cluster.
+
+4. KEB creates a cluster configuration in the Reconciler (except preview plan).
 
-4. Provisioner creates a new cluster and installs Kyma Runtime (for Kyma 1.x).
+5. Reconciler installs Kyma (except preview plan). 
 
-5. KEB creates a cluster configuration in the Reconciler (for Kyma 2.x).
+6. KEB creates Kyma resource (only for preview plan).
 
-6. Reconciler installs Kyma (for Kyma 2.x).  
+7. Lifecycle Manager manages Kyma modules (only for preview plan).
```

Comment:
maybe make 7. as a subpoint of 6 ? it's just an implication of the Kyma CR creation